RoboFile.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use DigipolisGent\Robo\Symfony\RoboFileBase;
+use DigipolisGent\Robo\Task\Deploy\Ssh\Auth\AbstractAuth;
+
+class RoboFile extends RoboFileBase
+{
+    protected function postSymlinkTask($worker, AbstractAuth $auth, $remote)
+    {
+        $currentProjectRoot = $remote['currentdir'] . '/..';
+        $collection = $this->collectionBuilder();
+        $parent = parent::postSymlinkTask($worker, $auth, $remote);
+        if ($parent) {
+            $collection->addTask($parent);
+        }
+        $collection->taskSsh($worker, $auth)
+            ->remoteDirectory($currentProjectRoot, true)
+            ->exec('composer symfony-scripts');
```

Comment:
Possibly (probably), until we have a symfony project that uses patches in the composer file (like in our Drupal projects), and this code would still need to be here (the symfony-scripts build the assets and whatnot)